docs/DeveloperGuide.adoc

```diff
@@ -295,35 +298,39 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |food blogger |keep a list of outstanding cafes |clear them in an orderly fashion.
 
-|`* * *` |user |add a new person |
+|`* * *` |student |keep a list of affordable eateries |save money
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |person with many food allergies |keep track of dishes I have ordered before |avoid ordering hazardous food
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |foodie |remember places |share this experience with my family and friends
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* *` |digital nomad |tag places with free power + wifi |sit there for extended periods of time
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`*` |working adult |view the menu beforehand |save time from thinking what to eat
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `EatMe` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Removing an eatery from todo
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to view his todo list
+2.  EatMe toggles to the todo mode
+3.  User requests to mark an eatery as visited in the list
+4.  EatMe prompts user to shift the eatery under the main mode (of visited places)
```

Comment:
May i confirm if EatMe will prompt the user to shift the eatery to main mode or will the action be based on a flag input by the user?